_component_design/radio-buttons.md

```diff
@@ -7,157 +7,134 @@ usage: >
-    <label>Select potential maintenance days</label>
-    <div class="ui buttons single">
-      <button class="ui button ds-btn-med-sec active">Mon</button><button class="ui button ds-btn-med-sec">Tues</button><button class="ui button ds-btn-med-sec">Wed</button><button class="ui button ds-btn-med-sec">Thurs</button><button class="ui button ds-btn-med-sec">Fri</button><button class="ui button ds-btn-med-sec">Sat</button><button class="ui button ds-btn-med-sec">Sun</button>
-    </div>
-    <div class="ui message subtext">
-      Here is some additional help text to explain these choices.
-    </div>
-  </div>
 </div>
 
-## Disabled
+## States
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Overview
+
+Form design best practices prohibit a radio button from existing in an error state. Avoid this situation.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Radio button states"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/radio-buttons/radio-states.svg)
+{% endfigure %}
+{% endcolumn %}
 
-<div class="ui form">
-  <div class="field">
-    <label>Select payment type</label>
-    <div class="ui buttons single">
-      <button class="ui button ds-btn-med-sec disabled active">Credit / Debit</button><button class="ui button ds-btn-med-sec disabled">Cash</button><button class="ui button ds-btn-med-sec disabled">Check</button>
-    </div>
-  </div>
 </div>
 
-## Error
-
-<div class="ui form error">
-  <div class="field error">
-    <label class="required">Select potential maintenance windows</label>
-    <div class="ui buttons single">
-      <button class="ui button ds-btn-med-sec">12<span class="smCaps"> AM</span> &ndash; 4<span class="smCaps"> AM</span></button><button class="ui button ds-btn-med-sec error active">4<span class="smCaps"> AM</span> &ndash; 8<span class="smCaps"> AM</span></button><button class="ui button ds-btn-med-sec">8<span class="smCaps"> AM</span> &ndash; 12<span class="smCaps"> PM</span></button><button class="ui button ds-btn-med-sec">12<span class="smCaps"> PM</span> &ndash; 4<span class="smCaps"> PM</span></button><button class="ui button ds-btn-med-sec">4<span class="smCaps"> PM</span> &ndash; 8<span class="smCaps"> PM</span></button><button class="ui button ds-btn-med-sec">8<span class="smCaps"> PM</span> &ndash; 12<span class="smCaps"> AM</span></button>
-    </div>
-    <div class="ui error message">
-      There is an error with the response to this question.
-    </div>
-  </div>
+
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Error state - expanded
+
+The question with the error receives an asterisk. The asterisk position floats to the left, in the margin, not in vertical alignment with the text. The section label should remain in its natural vertical alignment. 
```

Comment:
This needs to change per our decision to not munge the "required" asterisk with error states.

I've emailed you the new 2x PNG and SVG

Here's the new text:

Only the option with the error receives error styling.

The error icon is positioned to the left of the error message.

The error message should be 40-80 characters per line, and may wrap if needed.

Reply:
updated and new images included.